Readme.md

```diff
@@ -3,23 +3,63 @@
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E918)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
 
-* [Default.aspx](./CS/WebSite/Default.aspx) (VB: [Default.aspx](./VB/WebSite/Default.aspx))
-* [Default.aspx.cs](./CS/WebSite/Default.aspx.cs) (VB: [Default.aspx.vb](./VB/WebSite/Default.aspx.vb))
-<!-- default file list end -->
-# How to display progress information about server-side callback processing
+# ASP.NET Web Forms - How to display progress information about server-side callback processing
 <!-- run online -->
 **[[Run Online]](https://codecentral.devexpress.com/e918/)**
 <!-- run online end -->
 
+This example illustrates how you can inform your users about a server-side process when an operation initiated via a callback request takes much time.
+
+![](image.png)
+
+In this example, a callback panel sends a request to the server and performs a long server process. 
+
+Another callback panel sends a callback to the server every timer tick. On the server, a callback event handler gets the current operation progress and return this value to the client.
+
+## How to perform parallel callbacks
+
+This approach does not work if the Session state is used. The Session state blocks parallel execution and forces parallel requests to be executed one after another because the access to ASP.NET Session state is exclusive per session. 
```

Comment:
```suggestion
This approach does not work if the Session state is used. The Session state blocks parallel execution and forces parallel requests to be executed one after another because the access to the ASP.NET Session state is exclusive per session. 
```